build.cake

```diff
@@ -224,13 +224,19 @@ Task("RefDocs")
         var hugoExe = toolsHugoDirectory.CombineWithFilePath("hugo_0.13_windows_amd64.exe");
 
         var landingDirectory = docsDirectory.Combine("landing");
+        var landingPublicDirectory = landingDirectory.Combine("public");
+        CleanDirectory(landingPublicDirectory);
```

Comment:
CleanDirectory creates an empty directory if the directory does not exist. Either way, after CleanDirectory returns you end up with an empty directory.